tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+import ast
+
+import pycodestyle
+import pytest
+
+from flake8_import_order.flake8_linter import Linter
+
+
+def _get_linter(argv, code, filename):
+    parser = pycodestyle.get_parser('', '')
+    Linter.add_options(parser)
+    options, args = parser.parse_args(argv)
+    Linter.parse_options(options)
+    return Linter(
+        ast.parse(
+            code,
+            filename
+        ),
+        str(filename)
+    )
+
+
+@pytest.fixture
+def flake8_linter():
```

Comment:
You're right as it's used now there is no real benefit to it being a fixture. I had initially planned to do a bit more refactoring and use this fixture in all the places where a linter is used. I abandoned this however as I felt it was out of scope for this PR but then didn't think to revert this to a plain function.

If you want I can update the PR to use a plain function.